editor.planx.uk/src/components/Header.tsx

```diff
@@ -44,10 +44,10 @@ const BreadcrumbsRoot = styled(Box)(() => ({
   fontSize: 20,
 }));
 
-const BreadcrumbLink = styled(ReactNaviLink)(() => ({
+const StyledLink = styled(Link)(() => ({
   color: "#fff",
   textDecoration: "none",
-}));
+})) as typeof Link;
```

Comment:
Casting like this is required whenever we're using the `component={XXX}` prop (see line 194).